Dockerfile

```diff
@@ -1,16 +1,20 @@
-FROM python:3.8.1
+FROM python:3.8.1-slim
 
 ENV PYTHONUNBUFFERED 1
 
 EXPOSE 8000
 WORKDIR /app
 
+
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends netcat && \
+    rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
+
 COPY poetry.lock pyproject.toml ./
 RUN pip install poetry==1.0.* && \
     poetry config virtualenvs.create false && \
     poetry install --no-dev
 
-COPY . ./
+COPY docker/entrypoint.sh docker/entrypoint.sh
 
-CMD alembic upgrade head && \
-    uvicorn --host=0.0.0.0 app.main:app
+COPY . ./
```

Comment:
I think we don't need 2 `COPY` here. Since this entrypoint is not used for every launch of the image, then it seems to me that we can leave previous version with the default `CMD` command that started migrations and server.

Or as an alternative, then `entrypoint.sh` can be changed to work not only with the `host:port` format, but with the full DSN for Postgres also. And perhaps this is a better option, since it can work both with a container started using `compose` and with a container started using `docker` cli. Using this approach we can define `CMD ["docker/entrypoint.sh"]` in the `Dockerfile` and remove it from `docker-compose.yml`.

What do you think?

Reply:
In any case, we need to put the `CMD` instruction back to do anything at startup without `compose`.

Reply:
Let's keep it is